shell/functions.sh

```diff
@@ -80,6 +100,7 @@ build_k6_docker_command() {
             -e 'SPRYKER_TEST_RUN_ID=$testRunId' \
             -e 'SPRYKER_TEST_RUNNER_HOSTNAME=$(hostname)' \
             -e 'SPRYKER_TEST_ENVIRONMENT=$testEnvironment' \
+            -e 'SPRYKER_TEST_PATH=$relativePath' \
```

Comment:
SPRYKER_TEST_ENVIRONMENT we agreed to remove this, or it will be done in the next step (PR)? Other places also still have this value

Reply:
will be done in the next ticket